kafka/conn.py

```diff
@@ -487,7 +487,7 @@ def connect():
             self.connect()
             if self.connected():
                 return
-            timeout_at = time.time() + timeout
+            timeout_at = time.time() + (timeout_ms/1000)
```

Comment:
`(timeout_ms / 1000.0)` so we don't lose precision.

That said, I'd personally rather keep the calls as `timeout` instead of `timeout_ms` to reduce changes, but keep the config as `_ms` to stay consistent with the java configs. Not a huge deal though.